src/main.ts

```diff
@@ -39,37 +39,77 @@ let avgBarplot = new Barplot(
 $(window).on("resize", function () {
   avgBarplot.resize();
 });
-let data: DataPoint[];
-data = [
-  { name: "Victoria", value: "5", description: "Victoria" },
-  { name: "Vancouver", value: "10", description: "Vancouver" },
-  { name: "Edmonton", value: "20", description: "Edmonton" },
-  { name: "Calgary", value: "28", description: "Calgary" },
-  { name: "Saskatoon", value: "34", description: "Saskatoon" },
-  { name: "Regina", value: "38", description: "Regina" },
-  { name: "Winnipeg", value: "40", description: "Winnipeg" },
-  { name: "Windsor", value: "43", description: "Windsor" },
-  { name: "London", value: "54", description: "London" },
-  {
-    name: "Kitchener, Cambridge, Waterloo",
-    value: "55",
-    description: "Kitchener, Cambridge, Waterloo",
-  },
-  {
-    name: "St. Catharines, Niagara",
-    value: "60",
-    description: "St. Catharines, Niagara",
-  },
-  { name: "Hamilton", value: "70", description: "Hamilton" },
-  { name: "Toronto", value: "81", description: "Toronto" },
-  { name: "Montreal", value: "84", description: "Montreal" },
-  { name: "Sherbrooke", value: "84", description: "Sherbrooke" },
-  { name: "Quebec City", value: "94", description: "Quebec City" },
-  { name: "Halifax", value: "98", description: "Halifax" },
-  { name: "St. John's", value: "100", description: "St. John's" },
-];
-avgBarplot.plot(data, [0, 0], [100, 100]);
-avgBarplot.updatePlot(data);
+
+
+/*********************
+*** DRAW MEAN PLOT ***
+**********************/
+drawMeanPlot(avgBarplot);
+
+
+/**
+ * @brief   Renders the left-side plot with city averages, sorted descending.
+ * @param   avgBarplot : The barPlot to render.
+ * @returns N/A
+ */
+function drawMeanPlot(avgBarplot : Barplot) {
+  let countryData = Data.getSyncData();
+  let meanCountryData : DataPoint[] = [];
+  countryData.forEach((d) => addMeanCity(d));
+
+  function addMeanCity(cityData : DataModel) {
+    let citySummary: DataPoint = {
+      name:         cityData.name,
+      description:  cityData.name,
+      value:        (avgCityData(cityData.data) * 100).toString()
+    };
+    meanCountryData.push(citySummary);
+  }
+
+  meanCountryData.sort(compareDataPoint);
+
+  avgBarplot.plot(meanCountryData, [0, 0], [100, 100]);
```

Comment:
The [100, 100] you see here is a hardcoded max value, we could change it to [1,1] if you wanted to have the scale be between 0-1. This value should be inferred later on rather than hardcoded anyways

Reply:
Specifically is sets the min and max for the x-axis

Reply:
I'll see about using the max average value as a potential max x-value.